src/environments/environment.ts

```diff
@@ -3,7 +3,14 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  facebookId: '1782833315349267',
+  googleId:
+    '315082406739-imasn7gsabthac3uaer84oqmp1g2780p.apps.googleusercontent.com',
+  googleUrl:
```

Comment:
Hey @kelvinndmo, we have a base-url already created by Brain, Why don't we just use that.What do u think?

Reply:
sure